examples/mobile/HomeApp/webclip/video-service/css/style.css

```diff
@@ -0,0 +1,43 @@
+
+.app-video-control .ui-btn.ui-btn-flat.ui-btn-icon {
+    width: 56px;
+    height: 56px;
+    min-width: 56px;
+}
+.app-control-bar .ui-btn.ui-btn-contained {
+    font-size: 13px;
+    padding-top: 7px;
+    padding-bottom: 7px;
+}
+.app-control-bar {
+    display: flex;
+    justify-content: space-around;
+    padding: 10px;
+}
+.app-control-status {
+    padding: 20px;
+}
+.app-control-status .app-control-video-title {
+    font-size: 20px;
+    color: var(--text-color);
+}
+.app-control-status .app-control-video-subtitle {
+    font-size: 16px;
+    color: var(--text-secondary-color);
+}
+
+.ui-section-changer {
+	height: 100% !important;
+}
+.ui-section-changer-container {
```

Comment:
ui-section-changer-container was added to SectionChanger.js 
so shouldn't we add CSS also to TAU library?

Reply:
At the moment this cause issues with tests. These styles will be specific for this app.